streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -61,7 +61,7 @@ public StandbyTask(TaskId id,
         // initialize the topology with its own context
         this.processorContext = new StandbyContextImpl(id, applicationId, config, stateMgr, metrics);
 
-        initializeStateStores();
+        registerAndinitializeStateStores();
```

Comment:
@guozhangwang I need to do this because the constructor of `StandbyTask` requires the checkpointed offsets from the stage manager `stateMgr.checkpointedOffsets()`. The checkpointed offsets become available only after the initialisation process.